homeassistant/components/livisi/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "LIVISI Smart Home",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/livisi",
-  "requirements": ["aiolivisi==0.0.14"],
+  "requirements": ["aiolivisi==0.0.15"],
```

Comment:
Thanks for creating this independant PR.

> For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.

If you do not keep a changelog, please at least tag versions 0.0.14 and 0.0.15 on your repository, and add a link to the compare/diff